qt_gui_cpp/src/qt_gui_cpp_sip/CMakeLists.txt

```diff
@@ -52,6 +58,6 @@ if(sip_helper_FOUND)
     set(LIBQT_GUI_CPP_SIP_SUFFIX ${CMAKE_SHARED_LIBRARY_SUFFIX})
   endif()
 
-  install(FILES ${CATKIN_DEVEL_PREFIX}/${CATKIN_PACKAGE_PYTHON_DESTINATION}/libqt_gui_cpp_sip${LIBQT_GUI_CPP_SIP_SUFFIX}
-      DESTINATION ${CATKIN_PACKAGE_PYTHON_DESTINATION})
+  install(FILES ${CMAKE_CURRENT_SOURCE_DIR}../../lib/libqt_gui_cpp_sip${LIBQT_GUI_CPP_SIP_SUFFIX}
```

Comment:
Where is `${CMAKE_CURRENT_SOURCE_DIR}../..` supposed to point to? The variable doesn't have a trailing slash so following it with `..` seems wrong. Also why would the library be in the source space?

Reply:
Oh shoot, I fixed this in the testing branch. I'll bring this change in here too.

Reply:
Also, the source directory is currently where the makefile target is built.

https://github.com/ros-visualization/python_qt_binding/blob/a6cba30ed2c0e69e0bd5e0712281e6bb76081e01/cmake/sip_helper.cmake#L192